fetchUser(userId)
	.then(user => {
		return processUser(user);
	})
	.catch((error) => {
	handleError(error);
}